src/services/category.js

```
@@ -22,6 +23,56 @@ const categoryService = {
+  
+    return {
+      data: subjectNames,
+      total,
+      page,
+      limit,
+      totalPages: Math.ceil(total / limit)
+    };
+  },
+  
+  getCategoryById: async (categoryId) => {
+    const category = await Category.findById(categoryId).lean().exec();
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)